src/Enterspeed.Source.Sdk/Api/Models/IEnterspeedEntity.cs

```diff
@@ -72,4 +72,39 @@ public interface IEnterspeedEntity<T>
         /// </summary>
         T Properties { get; }
     }
+
+    internal interface IEnterspeedEntityV2<T>
```

Comment:
Don't know if the postfix `V2` is the best as it will also work with a potential version 3 of the ingest api. But the interface and class is marked as internal, so we can change it later without fearing for a breaking change.